content/en/docs/pipelines/multi-user.md

```diff
@@ -0,0 +1,188 @@
+experiment. Runs and recurring runs belong to their parent experiment's namespace.
+
+Pipeline runs run in user namespaces, so that users can leverage Kubernetes
+namespace isolation. For example, they can configure different secrets for other
+services in different namespaces.
+
+There's no multi-user isolation for pipeline definitions right now.
+
+### When using the UI
+
+When you visit the Kubeflow Pipelines UI from the Kubeflow dashboard, it only shows
+experiments, runs and recurring runs in your chosen namespace. Similarly, when
+you create resources from the UI, they also belong to the namespace you have
+chosen.
+
+You can select a different namespace to view resources in other namespaces.
+
+Other users cannot see your namespace without permission. They will be rejected by
+authorization checks in Kubeflow Pipelines API server.
+
+<!---
+#### Configuring Google Cloud permissions for namespace
+#### Granting Google Cloud permissions to UI Artifact Preview and Visualization
+
+For artifacts stored on Google Cloud Storage (GCS), Kubeflow Pipelines deploys
+artifact fetcher servers on user namespaces. The servers use `default-editor`
+Kubernetes service account, so that they are using the same Google service
+account as pipeline runs.
+-->
+
+### When using the SDK
+
+* <!-- this should be in GCP pipelines - auth doc --> how to authenticate to the public endpoint
+
+When calling SDK methods for experiments, you need to provide the additional
+namespace argument. Runs, recurring runs are owned by an experiment. They are
+in the same namespace as the parent experiment, so you can just call their SDK
+methods same as before.
+
+For example:
+
+```python
+import kfp
+client = kfp.Client(...)
+
+client.create_experiment(name='<Your experiment name>', namespace='<Your namespace>')
+print(client.list_experiments(namespace='<Your namespace>'))
+client.run_pipeline(
+    experiment_id='<Your experiment ID>', # Experiment determines namespace.
+    job_name='<Your job ID>',
+    pipeline_id='<Your pipeline ID>')
+print(client.list_runs(experiment_id='<Your experiment ID>'))
+print(client.list_runs(namespace='<Your namespace>'))
+```
+
+We also provide a helper method that saves a user namespace as default context
+in config file `$HOME/.config/kfp/context.json`. After setting a default
+namespace, the SDK methods default to use this namespace if no `namespace`
+argument provided.
+
+```python
+import kfp
+client = kfp.Client(...)
+
+# Note, this saves the namespace in `$HOME/.config/kfp/context.json`. Therefore,
+# You only need to call this once. The saved namespace context will be picked up
+# by other clients you use later.
+client.set_user_namespace(namespace='<Your namespace>')
+print(client.get_user_namespace())
+
+client.create_experiment(name='<Your experiment name>')
+print(client.list_experiments())
+client.run_pipeline(
+    experiment_id='<Your experiment ID>', # Experiment determines namespace.
+    job_name='<Your job ID>',
```

Comment:
nit: '<Your job name'> or <'The name of your job'>?
It matches the parameter name `job_name` better. Besides, when I read 'ID', even it's not named 'UID' or 'UUID', I subconsciously assume that it likely has to be unique. Here the `job_name` doesn't require uniqueness.